### DISCUSSION TITLE
Monitoring 404 with Google Analytics

### DISCUSSION THREAD

**Discussion by fdevans** [vuepress-theme-hope/vuepress-theme-hope#4549] [Q&A] [answered]
I am using the Google Analytics plugin to send data about page visits, but I don't see any metrics about the 404 page.  I'd really like to monitor activity hitting that page so I can track issues that our users might be running into.  Does anyone have any tips on how we might track 404s?

Since the app doesn't redirect it specifically to a new URL when something isn't found it seems like Google Analytics can't track the page load.

Using: Vuepress 2.0.0-rc.17  | vuepress-theme-hope 2.0.0-rc.57

**Comment by Mister-Hope** [comment, marked as answer]
please add further discussion in VuePress/ecosystem repo.
